libs/membership/website/src/lib/payment-amount/payment-amount-picker/payment-amount-picker.tsx

```diff
@@ -88,14 +86,33 @@ export const PaymentAmountPickerItem = forwardRef<HTMLButtonElement, PaymentAmou
 
 export const PaymentAmountPicker = forwardRef<HTMLInputElement, BuilderPaymentAmountProps>(
   (
-    {className, currency, amountPerMonthMin, amountPerMonthTarget, name, error, value, onChange},
+    {
+      className,
+      slug,
+      currency,
+      amountPerMonthMin,
+      amountPerMonthTarget,
+      name,
+      error,
+      value,
+      onChange
+    },
     ref
   ) => {
     const {
       elements: {TextField},
-      meta: {locale}
+      meta: {locale, siteTitle}
     } = useWebsiteBuilder()
 
+    const pickerItems = useMemo(() => {
+      switch (siteTitle) {
+        case 'Wnti':
+          return slug?.includes('spende') ? [10000, 15000, 20000] : [1000, 1500, 2000]
+        default:
+          return [1000, 1500, 2000]
+      }
+    }, [siteTitle, slug])
+
```

Comment:
This is a workaround until we (maybe?) allow to set the proposed amounts in the editor.